literature/wang_1996_ordered_6672/wang_1996_ordered_6672_p1_2b.yaml

```diff
@@ -0,0 +1,102 @@
+# This file describes a figure from a publication containing electrochemical data.
+# For the `echemdb/website` project, the source, curator and electrochemical system is mandatory.
+# Figure description is optional. It is created automatically by the cv module of svgdigitizer
+
+source:
+    version: 1
+    doi: 10.1021/jp952938y # only the number without preceeding doix or https
+    bib: wang_1996_ordered_6672 # AuthorName_YYYY_FirstWordTitle_pageNr. No prepositions. Fingers crossed that there is no duplicate.
+    figure: 2b # depending on the subfigure labels use 1, 1b, or 1_3
+    curve: # use a unique short description to identify the curve in the figure, i.e.. label, color, etc

```

Comment:
is this intentionally left blank?

Reply:
Since in Figure 2b, there is only one CV, I thought no further description would be necessary.